boards/arm/sensortile_box_pro/sensortile_box_pro.dts

```diff
@@ -0,0 +1,338 @@
+		 */
+		zephyr,console = &cdc_acm_uart0;
+		zephyr,shell-uart = &cdc_acm_uart0;
+
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_1: led_1 {
+			gpios = <&gpiof 6 GPIO_ACTIVE_HIGH>;
+			label = "User GREEN led";
+		};
+		red_led_1: led_2 {
+			gpios = <&gpioh 11 GPIO_ACTIVE_HIGH>;
+			label = "User RED led";
+		};
+		yellow_led_1: led_3 {
+			gpios = <&gpioh 12 GPIO_ACTIVE_HIGH>;
+			label = "User YELLOW led";
+		};
+		blue_led_1: led_4 {
+			gpios = <&gpiof 9 GPIO_ACTIVE_HIGH>;
+			label = "User YELLOW led";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		button1: button1 {
+			label = "User BT1";
+			gpios = <&gpioc 13 GPIO_ACTIVE_HIGH>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+		button2: button2 {
+			label = "User BT2";
+			gpios = <&gpioe 0 GPIO_ACTIVE_HIGH>;
+			zephyr,code = <INPUT_KEY_1>;
+		};
+	};
+
+	sd_card: sd-card {
+		compatible = "st,stile_sd_card";
+		en-sd-gpios = <&gpioh 10 (GPIO_ACTIVE_HIGH)>;
+		sel-sd-gpios = <&gpioh 8 (GPIO_ACTIVE_HIGH)>;
+	};
+
+	aliases {
+		led0 = &green_led_1;
+		led1 = &red_led_1;
+		led2 = &yellow_led_1;
+		led3 = &blue_led_1;
+		mcuboot-led0 = &blue_led_1;
+		mcuboot-button0 = &button1;
+		sw1 = &button1;
+		sw2 = &button2;
+		watchdog0 = &iwdg;
+		die-temp0 = &die_temp;
+		volt-sensor0 = &vref1;
+		volt-sensor1 = &vbat4;
+	};
+};
+
+&clk_hsi48 {
+	status = "okay";
+};
+
+&clk_lse {
+	status = "okay";
+};
+
+&clk_msis {
+	status = "okay";
+	msi-range = <4>;
+	msi-pll-mode;
+};
+
+&gpioi {
+	status = "okay";
+
+	/* switch sensor IMU bus from I2C1 to SPI2 */
+	mcu-sel-gpios {
+		gpio-hog;
+		gpios = <0 GPIO_ACTIVE_HIGH>;
+		output-low;
+	};
+};
+
+&pll1 {
+	div-m = <1>;
+	mul-n = <80>;
+	div-q = <2>;
+	div-r = <2>;
+	clocks = <&clk_msis>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll1>;
+	clock-frequency = <DT_FREQ_M(160)>;
+	ahb-prescaler = <1>;
+	apb1-prescaler = <1>;
+	apb2-prescaler = <1>;
+	apb3-prescaler = <1>;
+};
+
+&lptim1 {
+	clocks = <&rcc STM32_CLOCK_BUS_APB3 0x00000800>,
+		 <&rcc STM32_SRC_LSE LPTIM1_SEL(3)>;
+	status = "okay";
+};
+
+&usart1 {
+	pinctrl-0 = <&usart1_tx_pa9 &usart1_rx_pa10>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&usart3 {
+	pinctrl-0 = <&usart3_tx_pd8 &usart3_rx_pd9>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&uart4 {
+	pinctrl-0 = <&uart4_tx_pa0 &uart4_rx_pa1>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&spi1 {
+	pinctrl-0 = <&spi1_nss_pe12 &spi1_sck_pe13
+		     &spi1_miso_pe14 &spi1_mosi_pe15>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&spi2 {
+	pinctrl-0 = <&spi2_sck_pi1 &spi2_miso_pi2 &spi2_mosi_pi3>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	cs-gpios = <&gpioi 5 GPIO_ACTIVE_LOW>;
+
+	lsm6dsv16x: lsm6dsv16x@0 {
+		compatible = "st,lsm6dsv16x";
+		spi-max-frequency = <1000000>;
```

Comment:
You could use `DT_FREQ_M(1)` here.

Reply:
Ack!
(actually I will use DT_FREQ_M(10))

Reply:
thanks! yeah very hard to tell between 10M and 1M without the macro